.gitignore

```diff
@@ -1,7 +1,6 @@
 # Build subdirectories.
 /*build*
 !/build-aux
-!/build_msvc
```

Comment:
I'd keep this line. Otherwise, on Windows, build artifacts from pre-CMake branches, such as 28.x, will linger and be ignored by git.

Reply:
Isn't the point to ignore build artifacts though? Sure, they might linger but better than accidentally checking them into git.

Reply:
But build directories that aren't called `build_msvc` can still linger aorund in the same way. i think it's strange to keep this around but ok.

Reply:
@laanwj fwiw I agree with you, and think it should be dropped. Just couldn't be bothered having the argument.

Reply:
> I'm not sure if "accidentally checking them into git" is a concern we should worry about

i mean i personally use `git add .` sometimes to check all changes and additions in, relying that the `.gitignore` prevents any build artifacts from being checked in (because otherwise they're ignored all over the tree). So for some people it's probably a concern. Sure i check before pushing to github but sometimes it could result in extra work.

> @laanwj fwiw I agree with you, and think it should be dropped. Just couldn't be bothered having the argument.

Fair enough.

Reply:
I'm fine with any approach as well.

Reply:
> But build directories that aren't called `build_msvc` can still linger aorund in the same way. i think it's strange to keep this around but ok.

Other `build*` directories are created by the user explicitly, and the user is aware of them. However, the presence of the `build_msvc` directory might be unxepected.